schema/elasticsearch/visibility/versioned/v1/reindex.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+ES_SCHEME="${ES_SCHEME:-http}"
+ES_SERVER="${ES_SERVER:-127.0.0.1}"
+ES_PORT="${ES_PORT:-9200}"
+ES_USER="${ES_USER:-}"
+ES_PWD="${ES_PWD:-}"
+ES_VIS_INDEX_V0="${ES_VIS_INDEX_V0:-temporal-visibility-dev}"
+ES_VIS_INDEX_V1="${ES_VIS_INDEX_V1:-temporal-visibility-v1-dev}"
+
+REINDEX_CUSTOM_FIELDS="${REINDEX_CUSTOM_FIELDS:-true}"
+
+DIR_NAME=$(dirname "$0")
+# Convert multiline script to single line and remove repeated spaces.
+REINDEX_SCRIPT=$(tr -d "\n" < "${DIR_NAME}/reindex.painless" | tr -s " ")
+# Substitute envs in reindex.json (envsubst is not available in alpine by default).
+REINDEX_JSON=$(sed \
+  -e "s/\${REINDEX_SCRIPT}/${REINDEX_SCRIPT}/" \
+  -e "s/\${ES_VIS_INDEX_V0}/${ES_VIS_INDEX_V0}/" \
+  -e "s/\${ES_VIS_INDEX_V1}/${ES_VIS_INDEX_V1}/" \
+  -e "s/\${REINDEX_CUSTOM_FIELDS}/${REINDEX_CUSTOM_FIELDS}/" \
+  "${DIR_NAME}/reindex.json")
+
+echo "${REINDEX_JSON}" | jq
```

Comment:
is this just for debugging purposes? consider adding more details about what this is and what's happening or removing it.